mailet/standard/src/test/java/org/apache/james/transport/mailets/LogMessageTest.java

```diff
@@ -61,6 +62,7 @@
     @Before
     public void setup() {
         logger = mock(Logger.class);
+        when(logger.isInfoEnabled()).thenReturn(true);
```

Comment:
this commit "update unit tests" should not be a separated commit but a fixup, else it means some other commits where not building

Reply:
I'll rebase it. Sorry, correcting commit order for other people is always a pain. I will have a try.